javascript/javascript2/week3/warmupTask8.js

```diff
@@ -0,0 +1,19 @@
+const button = document.querySelector("button");
+
+let timeStampOld = new Date();
+const checkDoubleClick = () => {
```

Comment:
I would recommend to use the javascript API for getting an event on button double click. Although your code looks correct, you can't know for sure how much time have passed between two click events. see https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event